flames/selenium/README.md

```diff
@@ -0,0 +1,56 @@
+# Enable and Configure Selenium
+
+This section will guide you through getting Selenium integrated into the Devilbox.
+
+**See also**
+
+* [Docker Hub: Selenium](https://hub.docker.com/r/elgalu/selenium)
```

Comment:
I would also additionally link the documentation for that project: https://github.com/elgalu/docker-selenium/blob/master/README.md